asv_bench/benchmarks/frame_ctor.py

```diff
@@ -182,4 +182,21 @@ def time_frame_from_arrays_sparse(self):
         )
 
 
+class From3rdParty:
```

Comment:
we also likey don't have torch here in the benchmark suite

Reply:
could add for just the benchmark build

Reply:
where do we specify the deps for that?

Reply:
i think it's the default requirements-dev but adding torch may make this crazy

Reply:
ok.  i think this asv is nice to have but i wont lose sleep if it doesn't get run in the CI